dashboard/app/models/pd/application/principal_approval1920_application.rb

```diff
@@ -216,5 +216,38 @@ def additional_text_fields
         [:how_heard]
       ]
     end
+
+    # full_answers plus the other fields from form_data
+    def csv_data
+      sanitize_form_data_hash.tap do |hash|
+        additional_text_fields.each do |field_name, option, additional_text_field_name|
+          next unless hash.key? field_name
+
+          option ||= OTHER_WITH_TEXT
+          additional_text_field_name ||= "#{field_name}_other".to_sym
+          hash[field_name] = self.class.answer_with_additional_text hash, field_name, option, additional_text_field_name
+          hash.delete additional_text_field_name
+        end
+      end
+    end
+
+    def underrepresented_minority_percent
```

Comment:
This is already implemented in the principal approval base class. Still need it here?